specify/specify7
# Add picklist and `SpLocaleContainerItem` migration for collectionObjectType

Fixes #5172

- Creates picklist records for collectionObjectType
- Creates `SpLocaleContainerItem` records for collectionObjectType with a default picklist
  - This adds collectionObjectType as a relationship in Schema Config when choosing CollectionObject as base table
- Creates `SpLocaleItemStr` records to add field captions and descriptions for collectionObjectType in Schema Config


### Checklist

      and self-explanatory (or properly documented)

### Testing instructions



- Go to Schema Config
- Choose CollectionObject as base table
- [ ] Verify collectionObjectType is visible under relationships
- [ ] Verify collectionObjectType has a picklist `CollectionObjectType` assigned to it by default
- [ ] Verify collectionObjectType has a caption and description 
#### Add collectionObjectType to CollectionObject view definition
- Go to App Resources > Form Definition > Select the correct `.views` file for your current collection
- Go to the CollectionObject form definition
- Add collectionObjectType to the form definition as follows:
```xml
<cell type="label" labelfor="111"/>
<cell type="field" id="111" name="collectionObjectType" uitype="combobox"/>
```
- Go to Data Entry > CollectionObject
- [ ] Verify collectionObjectType appears as a picklist with all collectionObjectType names in the db
